src/python/pants/help/help_info_extracter.py

```diff
@@ -1106,9 +1109,16 @@ def get_option_help_info(self, args, kwargs):
         removal_hint = kwargs.get("removal_hint")
         choices = self.compute_choices(kwargs)
 
-        dest = Parser.parse_dest(*args, **kwargs)
+        dest = parse_dest(*args, **kwargs)
         # Global options have three env var variants. The last one is the most human-friendly.
```

Comment:
```suggestion
        # Global options have three env var variants
        # (eg. PANTS_GLOBAL_PANTS_WORKDIR, PANTS_PANTS_WORKDIR, PANTS_WORKDIR).
        # The last one is the most human-friendly.
```
maybe port these from the comment since its being deleted